sync/handlers.py

```diff
@@ -190,6 +190,14 @@ def __call__(self, git_gecko, git_wpt, body):
             logger.debug("No try push for SHA1 %s" % (sha1,))
             return
 
+        # Enforce the invariant that the taskgroup id is not set until
+        # the decision task is complete. This allows us to determine if a
+        # try push should have the expected wpt tasks just by checking if
+        # this is set
+        if result == "pending":
```

Comment:
It could also be "running" and "unscheduled"

* Maybe safer if we return if it's not exception, failed, completed.
* Would be nice to use `tc.PENDING`, etc., in this handler